tests/browser/test_create_user_context.py

```diff
@@ -52,3 +52,40 @@ def user_context(x):
                       'userContext': user_context_id
                   }],
                                               key=user_context)
+
+
+@pytest.mark.asyncio
+async def test_browser_create_user_context_proxy_server(
+        websocket, http_proxy_server):
+
+    # Localhost URLs are not proxied.
+    example_url = "http://example.com"
```

Comment:
What about `127.0.0.1`?

Reply:
it's the same. I think it skips proxy for anything that is pointing to local IP.

Reply:
It works for `192.x.x.x` IPs but we could figure out how to compute those later